docs/api/apiv3/components/schemas/storage_folder_write_model.yml

```diff
@@ -0,0 +1,13 @@
+# Schema: StorageFolderWriteModel
+---
+type: object
+required:
+  - name
+  - path
+properties:
+  name:
+    type: string
+    description: Folder name
```

Comment:
🟡 Being more specific woudl be nice, like "The name of the folder to be created" and for parent id something like "The unique identifier of the parent folder the new folder is created in"